tasks/xpack/elasticsearch-xpack.yml

```diff
@@ -20,4 +21,4 @@
 - name: Set elasticsearch.keystore Permissions
   become: yes
   file: state=file path={{ conf_dir }}/elasticsearch.keystore owner={{ es_user }} group={{ es_group }}
-  when: es_enable_xpack and es_xpack_features is defined and "security" in es_xpack_features and (es_version is version_compare('6.0.0', '>'))
+  when: es_enable_xpack
```

Comment:
I think we always want to do this even when using the oss version. But it was apparently already like this so its out of scope. I also think that the package properly creates the keystore with the correct permissions now but would need to confirm